### PR TITLE
Marking Video published which are posted on Hive

### PR DESCRIPTION
**Pull request by sag333ar**

### CODE REVIEW

File: scripts/publish_video/helper.js [hunk 1]
```diff
@@ -59,6 +59,68 @@ function hasPostingAuthority(account) {
   });
 }
 
+async function getHiveContentBeneficiaries(author, permlink) {
+  return new Promise((resolve, reject) => {
+    hive.api.getContent(author, permlink, function (err, result) {
```
Comment (vaultec81) [inline scripts/publish_video/helper.js:64]: preferably use dhive over hivejs

File: scripts/publish_video/helper.js [hunk 2]
```diff
@@ -360,6 +422,26 @@ async function shouldSkip(video) {
   }
 }
 
+async function isVideoAlreadyPostedOnHive(video) {
+  if (video.status === 'publish_manual') {
+    if (video.fromMobile === true ) {
+      try {
+        let doWeHavePostingAuthority = await hasPostingAuthority(video.owner);
```
Comment (vaultec81) [inline scripts/publish_video/helper.js:429]: How does checking posting authority have anything to do to being on chain? Plus it doesn't seem there are any checks for on chain here either

**Review by vaultec81 — CHANGES_REQUESTED** [review]

File: scripts/publish_video/helper.js (hunk 2 above)
Comment (sag333ar) [reply scripts/publish_video/helper.js:429]: my bad.

File: scripts/publish_video/helper.js (hunk 1 above)
Comment (sag333ar) [reply scripts/publish_video/helper.js:64]: Is this a blocker? I've already tested with `hive.api` and it is working as expected.
with dhive, I may have to rewrite function.

File: scripts/publish_video/helper.js (hunk 2 above)
Comment (vaultec81) [reply scripts/publish_video/helper.js:429]: @sag333ar What do you intend to do here going forward?

Comment (sag333ar) [reply scripts/publish_video/helper.js]: It's a function which was the result of copy-paste but forgot to remove it. I just updated code & removed this method.

File: scripts/publish_video/helper.js (hunk 1 above)
Comment (vaultec81) [reply scripts/publish_video/helper.js:64]: Not a blocker, but going forward dhive should be used over hivejs for most use cases.

Comment (sag333ar) [reply scripts/publish_video/helper.js:64]: Understood. We're anyway switching to new frontn-end & back-end. As a result, we'll have dhive everywhere.

**Pull request merged**